examples.md

```diff
@@ -200,40 +197,12 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
 
 ## Node - npm
 
-For npm, cache files are stored in `~/.npm` on Posix, or `%AppData%/npm-cache` on Windows. See https://docs.npmjs.com/cli/cache#cache
+For npm, cache files are stored in `~/.npm` on Posix, or `%AppData%/npm-cache` on Windows, but it's possible to use `npm config get cache` to find the path on any platform. See https://docs.npmjs.com/cli/cache#cache for more details.
```

Comment:
For the link use markdown. See [npmjs docs](https://docs.npmjs.com/cli/cache#cache) for more details.

Reply:
Fixed :+1: